front/src/views/Home.vue

```diff
@@ -3,30 +3,26 @@
         <v-container>
             <v-row dense>
                 <v-col cols="12">
-                    <v-card color="green darken-3" dark>
-                        <v-card-title class="headline"
-                            >Welcome,
-                            {{ user.displayName || user.email }}</v-card-title
-                        >
+                    <v-card color="#49868C" dark>
```

Comment:
can you provide screenshot for these new colors please ?

Reply:
Ups my bad i commit this test :/ 
![image](https://user-images.githubusercontent.com/3717296/77622193-19385880-6f3e-11ea-9dd4-530e014d39bc.png)

Reply:
And i try to put the iframe in a card too